pkg/privilege/privileges/ldap/test/ca.crt

```diff
@@ -1,31 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIFZTCCA02gAwIBAgIUZ2hQOFVvjuAHrC8Tv+5dnwPGvF0wDQYJKoZIhvcNAQEL
```

Comment:
why do we need to change privileges files here?

Reply:
Because  V7.5 branch has merged one pr https://github.com/pingcap/tidb/pull/60747 fixed CI problem。